services/QuillLMS/app/services/admin_diagnostic_reports/construct_query_payload.rb

```diff
@@ -0,0 +1,59 @@
+# frozen_string_literal: true
+
+module AdminDiagnosticReports
+  class ConstructQueryPayload < ApplicationService
+    attr_reader :user_id, :shared_filters_name, :specific_filters_name
+
+    DEFAULT_TIMEFRAME = 'this-school-year'
+    DEFAULT_AGGREGATION = 'grade'
+    DEFAULT_DIAGNOSTIC_ID = 1663
+
+    def initialize(user_id, shared_filters_name, specific_filters_name)
+      @user_id = user_id
+      @shared_filters_name = shared_filters_name
+      @specific_filters_name = specific_filters_name
+    end
+
+    def run
```

Comment:
This comment should be tagged [Optional], since it's not currently enforced by our Rubocop or our style guide. 

Rubocop's style guide considers it "opt-in": https://github.com/rubocop/ruby-style-guide?tab=readme-ov-file#endless-methods

There's also a Cop available: https://www.rubydoc.info/gems/rubocop/RuboCop/Cop/Style/EndlessMethod

FWIW I like the Endless Method syntax in most cases, but want to allow for developer discretion in cases we don't necessarily anticipate right now.

Reply:
As we've discussed in engineering meetings, the endless syntax is **always** optional.  I only point it out as a stylistic suggestion.

Reply:
But I think your point about explicitly tagging as optional is correct.